src/indev/lv_indev.c

```diff
@@ -1314,6 +1328,36 @@ static void indev_proc_release(lv_indev_t * indev)
     }
 }
 
+static void indev_proc_pointer_diff(lv_indev_t * indev)
+{
+    lv_obj_t * obj = indev->pointer.last_pressed;
+    if(obj == NULL) return;
+    if(indev->pointer.diff == 0) return;
+
+    indev_obj_act = obj;
+
+    bool editable = lv_obj_is_editable(obj);
+
+    if(editable) {
+        int32_t sensitivity = indev->rotary_adjust_divider;
+        int32_t diff = indev->pointer.diff / sensitivity;
+        send_event(LV_EVENT_ROTARY, &diff);
+    }
+    else {
+
+        int32_t sensitivity = indev->rotary_scroll_sensitvity;
```

Comment:
Sensitivity could be a value specified per widget. For example an that has large items may want to scroll faster.

Reply:
I think both are needed in this case. The number of clicks per turn is indev specific so you should be able to set up a unified behavior regardless to the rotary characteristics. 

We can add a `rotary_scroll_sensitivity;` style property.

Do we need a widget specific setting [here ](https://github.com/lvgl/lvgl/pull/5057/files/849512eb67de9449e9efa0fabe1d4bdb8ebcd1da#diff-35a05ebbf5595318ed115bcabc7f70e6ae0b4f1f3ace3629e154a7434fe680f1R1342)too? E.g. to adjust a slider with larger range faster.

Reply:
> We can add a `rotary_scroll_sensitivity;` style property.

Agree. 



> Do we need a widget specific setting [here ](https://github.com/lvgl/lvgl/pull/5057/files/849512eb67de9449e9efa0fabe1d4bdb8ebcd1da#diff-35a05ebbf5595318ed115bcabc7f70e6ae0b4f1f3ace3629e154a7434fe680f1R1342)too? E.g. to adjust a slider with larger range faster.

Yes. Application can adjust the speed as they want by style.

Reply:
> We can add a rotary_scroll_sensitivity; style property.

Agree.

> Do we need a widget specific setting here too? E.g. to adjust a slider with larger range faster.

Yes. In this way, application can adjust the speed as they want by style.